tests/voq/test_fabric_cli_and_db.py

```diff
@@ -0,0 +1,288 @@
+
+            # Isolate the port
+            cmd = "sudo config fabric port isolate {} {}".format(localPort, asicNamespaceOption)
+            cmd_output = duthost.shell(cmd, module_ignore_errors=True)
+            stderr_output = cmd_output["stderr"]
+            pytest_assert(
+                  len(stderr_output) <= 0, "command: {} failed, error: {}".format(cmd, stderr_output))
+
+            # Check the isolateStatus in CONFIG_DB
+            cmd = "sonic-db-cli {} CONFIG_DB hget 'FABRIC_PORT|Fabric{}' isolateStatus".format(asicNamespaceOption,
+                                                                                               localPort)
+            cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")
+            tokens = cmd_output[0].split()
+            pytest_assert(
+                  len(tokens) > 0,
+                  "FABRIC_PORT|Fabric{} isolateStatus not found in CONFIG_DB".format(localPort))
+            isolateStatus = tokens[0]
+            pytest_assert(
+                  isolateStatus == "True",
+                  "Port {} CONFIG_DB initial isolateStatus is True, expected False".format(localPort))
+
+            # Check the isolateStatus in APPL_DB
+            cmd = "sonic-db-cli {} APPL_DB hget 'FABRIC_PORT_TABLE:Fabric{}' isolateStatus".format(asicNamespaceOption,
+                                                                                                   localPort)
+            cmd_output = duthost.shell(cmd, module_ignore_errors=True)["stdout"].split("\n")
+            tokens = cmd_output[0].split()
+            pytest_assert(
+                  len(tokens) > 0,
+                  "FABRIC_PORT_TABLE:Fabric{} isolateStatus not found in APPL_DB".format(localPort))
+            isolateStatus = tokens[0]
+            pytest_assert(
+                  isolateStatus == "True",
+                  "Port {} APPL_DB initial isolateStatus is True, expected False".format(localPort))
+
+            # If the port was originally not isolsated then restore it
+            if originalIsolateStatus == "False":
+                cmd = "sudo config fabric port unisolate {} {}".format(localPort, asicNamespaceOption)
+                cmd_output = duthost.shell(cmd, module_ignore_errors=True)
+                stderr_output = cmd_output["stderr"]
+                pytest_assert(
+                      len(stderr_output) <= 0,
+                      "command: {} failed, error: {}".format(cmd, stderr_output))
+
+
+# This test iterates over the fabric links on each asic
+# on the supervisor.
+# It isolates and unisolates each fabric link. Each time the
+# state of the link is changed the state is checked in both
+# CONFIG_DB and APPL_DB.
+# The values in CONFIG_DB are updated by the fabric CLI commands
+# and the values in APPL_DB are updated by the fabric manager
+# daemon.
+
+
+def test_fabric_cli_isolate_supervisor(duthosts, enum_supervisor_dut_hostname):
```

Comment:
add condtion to run this test for Voq T2 chassis only

Reply:
done , added 

pytestmark = [
    pytest.mark.topology('t2')
]